### PR TITLE
Track-Field/item Service

### PR DESCRIPTION
**Pull request by luericlueric**
Added Track-Field/item Service. 
allow querying by `id`, `name`, `issingle`, `istrack`, and `heatsize`

### CODE REVIEW

File: src/services/track-field/item/item.schema.ts [hunk 1]
```diff
@@ -5,23 +5,32 @@ import type { Static } from '@feathersjs/typebox'
...
 import { dataValidator, queryValidator } from '../../../validators'
+import { toLowerCaseProperty } from '../../../utilities/property-name-converter';
 
 // Main data model schema
 export const tfItemSchema = Type.Object(
   {
     id: Type.Number(),
-    text: Type.String()
+    name: Type.String(),
+    issingle: Type.Number(),
```
Comment (umliuxin) [inline src/services/track-field/item/item.schema.ts]: boolean?

File: src/services/track-field/item/item.schema.ts [hunk 2]
```diff
@@ -5,23 +5,32 @@ import type { Static } from '@feathersjs/typebox'
...
+    istrack: Type.Number(),
```
Comment (umliuxin) [inline src/services/track-field/item/item.schema.ts]: boolean?

**Comment by luericlueric** [comment]
image

File: src/services/track-field/item/item.schema.ts (hunk 1 above)
Comment (luericlueric) [reply src/services/track-field/item/item.schema.ts]: [9f8e8f0](https://github.com/umliuxin/svcsa-tf-api/pull/6/commits/9f8e8f00be84c651864972dde8ff17c9b3e9345a)

File: src/services/track-field/item/item.schema.ts (hunk 2 above)
Comment (luericlueric) [reply src/services/track-field/item/item.schema.ts]: [9f8e8f0](https://github.com/umliuxin/svcsa-tf-api/pull/6/commits/9f8e8f00be84c651864972dde8ff17c9b3e9345a)

**Review by umliuxin — APPROVED** [review]

**Pull request merged**